### PR TITLE
#274 functionality partial implementation (simplified team_switch file changes)

### PR DESCRIPTION
**Pull request by Jazzer008**
Changes visualized [here](https://github.com/ash47/LegendsOfDota/issues/274#issuecomment-222333458).

### CODE REVIEW

File: src/panorama/scripts/custom_game/team_switch.js [hunk 1]
```diff
@@ -0,0 +1,104 @@
+"use strict";
+
+var customTeamAssignments = {};
+var active = false;
+
+GameEvents.Subscribe( "player_team", GetTeamInfo);
```
Comment (ash47) [inline src/panorama/scripts/custom_game/team_switch.js]: Doto themed code, why you do this? :(

The coding scheme that has been used through out the project is that strings are defined using single quotes (') and that function calls do not have spaces inbetwene the backets and the arguments, and all functions and variables use camel case, ie:

( "player_team", GetTeamInfo);

should be:

('player_team', getTeamInfo)

LoD has a well defined style, a style it has maintained for 2.5 years now, I know the offical dota coding style is different, however, LoD has existed for LONGER than the official modding framework has existed, so, I intend to stick to it.

File: src/panorama/styles/custom_game/team_switch.css [hunk 2]
```diff
@@ -0,0 +1,155 @@
+.TeamSwitchRoot
```
Comment (ash47) [inline src/panorama/styles/custom_game/team_switch.css]: sorry to be annoying with this, however, the general lod naming scheme is that we use camel case, meaning that:

TeamSwitchRoot

becomes

teamSwitchRoot

**Comment by Jazzer008** [comment]
@ash47 Sure thing, I'll change it all.

**Comment by Jazzer008** [comment]
@ash47 Ok, all done.

**Pull request closed without merging**